internal/cluster/transition.go

```diff
@@ -159,6 +159,9 @@ func createClusterCompletionStatusInfo(log logrus.FieldLogger, cluster *common.C
 		return statusInfoInstalled
 	}
 
+	msg := fmt.Sprintf("Cluster %s is installed but degraded due to failed OLM operators", cluster.Name)
+	eventHandler.AddEvent(ctx, *cluster.ID, nil, models.EventSeverityWarning, msg, time.Now())
```

Comment:
I think that this code should be in https://github.com/openshift/assisted-service/blob/12d033890f5ff671d78c990e32af7071f7bc1df7/internal/cluster/cluster.go#L1097 after we update the cluster status and not here. @YuviGold WDYT?

Reply:
Actually, I think that's fine. Since this is the only place that defines a cluster as degraded.